docs/changelog/118366.yaml

```diff
@@ -0,0 +1,24 @@
+pr: 118366
+summary: |-
+  Configuring a bind DN in an LDAP or Active Directory (AD) realm without a corresponding bind password
+  will prevent node from starting
+area: Authentication
+type: breaking
+issues: []
+breaking:
+  title: -|
+    Configuring a bind DN in an LDAP or Active Directory (AD) realm without
+    a corresponding bind password will prevent node from starting
+  area: Cluster and node setting
+  details: -|
+    For LDAP or AD authentication realms, setting a bind DN (via the
+    `xpack.security.authc.realms.ldap.*.bind_dn` realm setting) without a
+    bind password is a misconfiguration that may prevent successful
+    authentication to the node. Nodes will fail to start if a bind DN is specified without a password.
+  impact: -|
+    If you have a bind DN configured for an LDAP or AD authentication
+    realm, set a bind password for
+    [LDAP](https://www.elastic.co/guide/en/elasticsearch/reference/current/ldap-realm.html#ldap-realm-configuration)
```

Comment:
If I'm not mistaken, instead of absolute URLs, I think we should make these links relative and leverage the `{ref}` variable. Meaning, we should replace them with:
- `{ref}/ldap-realm.html#ldap-realm-configuration[LDAP]`
- `{ref}/active-directory-realm.html#ad-realm-configuration[Active Directory]`

Reply:
Nice, if that works, absolutely -- changelogs are yaml as opposed to asciidoc so I'm not positive the same rules apply. I asked in the ES docs channel -- lets see!